src/ophys_etl/modules/segmentation/utils/roi_utils.py

```diff
@@ -301,6 +340,15 @@ def convert_roi_keys(roi_list: List[Dict]) -> List[Dict]:
     return new_list
 
 
+def roi_list_from_deserialized(rois: List[dict]) -> List[OphysROI]:
```

Comment:
Should this method (and the `roi_list_from_file` method you inherited) be named `ophys_roi_list_from_deserialized` so that it is more explicit which class is being returned? Or will that be too long?

Reply:
done